### PR TITLE
docs(accordion): decompose Simple accordion

### LINKED ISSUE #3616: docs(accordion): decompose Simple accordion
Decompose `Simple accordion` section (related to #3128)

### PR DESCRIPTION
**Pull request by artolshansky**
Decompose to Simple, With open event, With custom HTML

close #3616

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+accordion/accordion-section.list.ts [hunk 1]
```diff
@@ -38,6 +39,14 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/basic/basic.html'),
         outlet: DemoAccordionBasicComponent
       },
+      {
+        title: 'With opening event',
+        anchor: 'open-event',
+        description: `<p>Accordion with opening (<code>isOpenChange</code>) event listener.</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+accordion/accordion-section.list.ts]: remove `opening`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by EvilAlexei — CHANGES_REQUESTED** [review]
And where is demo with custom html?

File: demo/src/app/components/+accordion/accordion-section.list.ts [hunk 2]
```diff
@@ -38,6 +40,21 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/basic/basic.html'),
         outlet: DemoAccordionBasicComponent
       },
+      {
+        title: 'With opening event',
+        anchor: 'open-event',
+        description: `<p>Accordion with (<code>isOpenChange</code>) event listener.</p>`,
```
Comment (EvilAlexei) [inline demo/src/app/components/+accordion/accordion-section.list.ts]: Remove brackets.

File: demo/src/app/components/+accordion/accordion-section.list.ts [hunk 3]
```diff
@@ -38,6 +40,21 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/basic/basic.html'),
         outlet: DemoAccordionBasicComponent
       },
+      {
+        title: 'With opening event',
```
Comment (EvilAlexei) [inline demo/src/app/components/+accordion/accordion-section.list.ts]: Rename to Event isOpenChange

**Review by EvilAlexei — CHANGES_REQUESTED** [review]
Rename `Simple accordion` to `Basic`

**Review by EvilAlexei — DISMISSED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: demo/src/app/components/+accordion/accordion-section.list.ts [hunk 4]
```diff
@@ -30,14 +32,29 @@ export const demoComponentContent: ContentSection[] = [
...
+        title: 'Basic',
+        anchor: 'basic-accordion',
         description: `<p>Click headers to expand/collapse content that is broken into logical sections, much
           like tabs.</p>`,
         component: require('!!raw-loader?lang=typescript!./demos/basic/basic'),
         html: require('!!raw-loader?lang=markup!./demos/basic/basic.html'),
         outlet: DemoAccordionBasicComponent
       },
+      {
+        title: 'Event isOpenChange',
```
Comment (valorkin) [inline demo/src/app/components/+accordion/accordion-section.list.ts]: worst name ever :)

File: demo/src/app/components/+accordion/demos/custom-html/custom-html.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<accordion>
+  <accordion-group #group>
```
Comment (valorkin) [inline demo/src/app/components/+accordion/demos/custom-html/custom-html.html]: `#group` is not used and duplicated,
clean up things you don't need to have in demos
they should be simple and minimal
not confusing

**Review by valorkin — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested. looks good

**Review by valorkin — APPROVED** [review]

**Review by valorkin — DISMISSED** [review]

**Pull request merged**